ui/packages/lib/src/components/nav_drawer/NavDrawer.js

```diff
@@ -53,7 +58,9 @@ export const NavDrawer = ({ homeUrl = "/", docLinks }) => {
 
         callback: () =>
           !children || !projectId
-            ? navigate(projectId ? `${a.href}/projects/${projectId}` : a.href)
+            ? window.location.href = !!projectId
```

Comment:
I'm curious about this line. At first, I thought it should be a condition expression for the ternary operator, but it seems you are assigning value to `window.location.href` by using one `=`. Is it another JS magic that I don't know? Please enlighten me.

Reply:
Which part is magical here? :) yeah, I'm assigning a value (conditional) to `window.location.href` and with that you can sort of programatically trigger a click on a link with assigned href value. 
Trick is that `navigate` comes from `@reach/router` and it navigates inside a single page app (SPA) (without triggering a full refresh of the page), so I need to emulate a click on a link to completely refresh the page (so another SPA can be downloaded)